test/lib/helper.rb

```diff
@@ -210,7 +210,7 @@ def with_server(bin, addr, &block)
           'KEY_CLAIM_TOKEN' => 'first-very-long-token=302:second-very-long-token=302',
           'DATABASE_URL' => DATABASE_URL,
         },
-        bin, STDERR => File.open('/dev/null')
+        bin, STDERR => :out
```

Comment:
My googling has failed me to find what :out means in Ruby, I'm sure it's fine but if you can enlighten me that'd be great

Reply:
It just outs the Go servers logs into the same output as the test output, this was more for debugging and I will clean it up once I can figure out what is going on.